protocol/message_persistence.go

```diff
@@ -552,6 +552,16 @@ func (db sqlitePersistence) messageByID(tx *sql.Tx, id string) (*common.Message,
 	return getMessageFromScanRows(db, rows)
 }
 
+func (db sqlitePersistence) messageByAlbumID(albumID string) ([]*common.Message, error) {
+	query := db.buildMessagesQuery("WHERE m1.album_id = ?")
```

Comment:
this will need an index, also I think we should query by local_chat_id and album_id, since album_id it's in theory not unique, and this will allow in theory someone to control whether their messages are going to be marked as seen/unseen in a different chat